src/main/resources/application.yaml

```diff
@@ -927,6 +927,15 @@ govnotify:
           sms: 24dd4193-554b-40be-a0b3-56b0f80e37ab
         respondent:
           email: 6ebfdf22-9d9c-468e-a7cb-0fa84a7986e9
+    updateTribunalDecision:
+      rule31:
+        legalrep:
+          beforeListing:
+            email: da39849d-58cd-4e91-ab79-ea3083acd753
```

Comment:
We don't need to show Listing reference anymore, so we don't need templates here